### DISCUSSION TITLE
Overlap of math content between two `enum.item`s

### DISCUSSION THREAD

**Discussion by Andrew15-5** [typst/typst#2175] [Q&A]
```typ
+ some text here $alpha_1, ..., alpha_((nu - 1) / 2)$;
+ $V = -2ln(product_(i=0)^((nu/2) - 1) a i alpha_i)$
```

![image](https://github.com/typst/typst/assets/37143421/36fbcda9-f097-492a-89ce-86b4a4c2fa7f)

Is this considered a bug, or does the author have to add additional vertical spacing manually? If it's not a bug, please, provide an example of how to fix this issue.

**Comment by Dherse** [comment]
That looks like a bug to me.

In the meantime, you can increase the spacing: `set enum(spacing: 5pt)` or something like that (adjust to your needs of course).

**Reply by Andrew15-5** [reply]
Yeah, about that... I think there is another bug, since it doesn't work:

```typ
#set enum(spacing: 1000pt)
+ some text here $alpha_1, ..., alpha_((nu - 1) / 2)$;
+ $V = -2ln(product_(i=0)^((nu/2) - 1) a i alpha_i)$
```

![image](https://github.com/typst/typst/assets/37143421/c93d8b2f-b2b1-4022-be8f-791597bbe5f2)

**Reply by Andrew15-5** [reply]
I can add `  #v(0.1em)` on the next after first `enum.item` which does work (but very ugly).

**Reply by laurmaedje** [reply]
Enum spacing is only used for wide enums with a line break. Tight enums use leading. The overlap is a known issue: #1028

**Reply by Andrew15-5** [reply]
I tried changing `par(leading)` and it works (in every case). `enum(tight)` doesn't change anything. If `enum(tight: false)`, it should use `enum(spacing)`, but it doesn't.
